core/src/main/scala/org/apache/spark/FutureAction.scala

```diff
@@ -58,7 +58,7 @@ trait FutureAction[T] extends Future[T] {
    *
    * @param atMost maximum wait time, which may be negative (no waiting is done), Duration.Inf
    *               for unbounded waiting, or a finite positive duration
-   * @throws Exception exception during action execution
+   * @note Throws `Exception` exception during action execution
```

Comment:
Yes.. It seems fine although it does not have a link.

- Scaladoc
  <img width="459" alt="2017-01-30 10 34 38" src="https://cloud.githubusercontent.com/assets/6477701/22424970/53377738-e73c-11e6-9aac-ac3cabc57e42.png">

- Javadoc
  <img width="362" alt="2017-01-30 10 35 36" src="https://cloud.githubusercontent.com/assets/6477701/22425007/711806be-e73c-11e6-9cc2-580682cabc21.png">

Let me revert back those changes.